lesson07/script.js

```diff
@@ -0,0 +1,44 @@
+const loading = document.getElementById('js-loading');
+
+function createLoading(){
+    const loadingImg = document.createElement('img');
+    loadingImg.src = "loading-circle.gif";
+    loadingImg.alt = "ローディング画像";
+    loading.appendChild(loadingImg);
+}
+
+function removeLoading(){
+    loading.remove();
+}
```

Comment:
```suggestion
function createLoading() {
  const loading = document.getElementById("js-loading");
  const loadingImg = document.createElement("img");
  loadingImg.src = "loading-circle.gif";
  loadingImg.alt = "ローディング画像";
  loading.appendChild(loadingImg);
}

function removeLoading() {
  document.getElementById("js-loading");.remove();
}

```
nits
It's very neatly organized for each function, so you may be able to put the first variable in the function.

Reply:
Thank you for your comment!
I agree with you.
It's better to put variables together in a function.